lib/command/run.rb

```diff
@@ -434,6 +433,8 @@ def wait_for_job_status_and_log(logs_pipe) # rubocop:disable Metrics/MethodLengt
     end
 
     def print_detached_commands
+      return unless replica
```

Comment:
Just to prevent printing the detached commands if pressing `Ctrl + C` too early (before the replica is created).